.github/workflows/issue-routing-helper.yml

```diff
@@ -0,0 +1,37 @@
+name: Issue Routing Helper
+on:
+  issues:
+    types: [labeled]
+env:
+  # Use GH_RELEASE_PAT as github-actions bot is not allowed to ping teams
+  GH_TOKEN: ${{ secrets.GH_RELEASE_PAT }}
+  GH_REPO: ${{ github.repository }}
+jobs:
+  route:
+    runs-on: ubuntu-latest
+    if: "startsWith(github.event.label.name, 'Team: ')"
+    steps:
+      - name: "Ensure a single 'Team: *' label with 'Status: Untriaged'"
+        run: |
+          labels_to_remove=$(gh api --paginate "/repos/$GH_REPO/labels" -q '[.[].name | select((startswith("Team: ") or startswith("Status: ")) and . != "${{ github.event.label.name }}" and . != "Status: Untriaged")] | join(",")')
+          gh issue edit ${{ github.event.issue.number }} --remove-label "$labels_to_remove" --add-label '${{ github.event.label.name }},Status: Untriaged'
+      - name: "Mention/ping assigned team for triage"
+        run: |
+          # Get team label mention name:
+          mention_target=$(gh api "/repos/$GH_REPO/labels/${{ github.event.label.name }}" -q '.description')
+          mention_slug=$(gh api "/orgs/getsentry/teams/$mention_target" -q .slug || true)
+          if [[ -z "$mention_slug" ]]; then
+            echo "Couldn't find team mention from slug, trying the label name"
```

Comment:
Should we make the default, and only try the label description if we fail? Ideally I think all of the team labels and team names should line up.